server/strategies/localSignup.js

```diff
@@ -1,6 +1,6 @@
 import UserModel from '../models/UserModel'
 
-export default async function (req, username, password, done) {
+export default async function (req, username, _password, done) {
```

Comment:
this strategy name suggests sign up but the method itself looks for an existing user. Is this named correctly? Should it create a user instead of finding one? Also can we remove the double negative in the ternary on line 10?

Reply:
this line..?
```javascript
return !user ? done(null, false, req.body) : done(null, true, null)
```